lib/FeedExpander.php

```diff
@@ -48,7 +48,9 @@ protected function collect_RSS_1_0_data($rssContent, $maxItems){
 		$this->load_RSS_2_0_feed_data($rssContent->channel[0]);
 		foreach($rssContent->item as $item){
 			debugMessage('parsing item ' . var_export($item, true));
-			$this->items[] = $this->parseItem($item);
+			if (null !== $this->parseItem($item)) {
+				$this->items[] = $this->parseItem($item);
```

Comment:
This is calling the same function as in the if-statement. Since `parseItem` could potentially load additional data during execution its result should be put into a variable instead of calling it twice.